reverse_engineering/helpers/utils.js

```diff
@@ -98,6 +98,42 @@ const getTemplateDocByJsonSchema = schema => {
 	}, {});
 };
 
+/**
+ *
+ * @param {Array<number | string | boolean>} list
+ * @returns {number | string | boolean}
+ */
+const getMostFrequentValueInList = list => {
```

Comment:
Based on the naming conventions used in the function, I propose naming it getMostFrequentItemInList.